tests/testthat/test-create-plots-from-excel.R

```diff
@@ -855,10 +855,15 @@ test_that("It returns an empty DataCombined when no data is available", {
         "exportConfiguration" = exportConfigurationDfLocal
       ), path = file.path(tempDir, "Plots.xlsx"), )
 
-      dataCombined <- createDataCombinedFromExcel(
-        file = file.path(tempDir, "Plots.xlsx"),
-        stopIfNotFound = FALSE
-      )
+      # Warnings are suppressed because they are expected but not relevant for
```

Comment:
Nice, we have tons of warnings in the tests, should be good to suppress them when they are actually expected.